cmd/vic-machine/create/create.go

```diff
@@ -1257,12 +1250,29 @@ func (c *Create) Run(cliContext *cli.Context) (err error) {
 	log.Info("")
 
 	executor := management.NewDispatcher(ctx, validator.Session, vchConfig, c.Force)
+
 	if err = executor.CreateVCH(vchConfig, vConfig); err != nil {
 		executor.CollectDiagnosticLogs()
 		log.Error(err)
 		return err
 	}
 
+	// after create appliance, use timeout context to wait ip address
+	timeoutctx, cancel := context.WithTimeout(context.Background(), c.Timeout)
+	defer cancel()
+	defer func() {
+		if timeoutctx.Err() == context.DeadlineExceeded {
+			//context deadline exceeded, replace returned error message
+			err = errors.Errorf("Creating VCH exceeded time limit of %s. Please, increase timeout using --time in case of slow network bandwidth or busy vSphere target.", c.Timeout.String())
```

Comment:
no comma after Please
agree with @andrewtchin it should be --timeout

Maybe this for the text:
Please increase the timeout using --timeout to accommodate for a slow network connection or a busy vSphere target.